doc/source/ray-core/examples/plot_pong_example.ipynb

```diff
@@ -31,7 +31,7 @@
     "To run the application, first install some dependencies.\n",
     "\n",
     "```bash\n",
-    "pip install gymnasium[atari] gym==0.26.2\n",
+    "pip install gymnasium[atari]==0.28.1\n",
```

Comment:
I'm curious, why did this change?

Reply:
gymnasium 0.26.2 did not come with ALE prepackaged I believe; that's why we were doing this as well as the workaround where we we using ALE environments from gym through the gymnasium->gym wrapper e.g. `gymnasium.make("GymV26Environment-v0", env_id="ALE/Pong-v5")`.